marketplace/accounts/views.py

```diff
@@ -0,0 +1,70 @@
+from rest_framework import viewsets
+from marketplace.accounts.serializers import ProjectAuthorizationSerializer, UserPermissionSerializer
+
+from .models import ProjectAuthorization
+
+from rest_framework.response import Response
+from rest_framework import status
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+
+class UserViewSet(viewsets.ViewSet):
+    def get_serializer(self, *args, **kwargs):
+        return UserPermissionSerializer()
+
+    def create(self, request):
+
+        if not request.user.has_perm("accounts.can_communicate_internally"):
+            return Response({"error": "Not Allowed"}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Use the ValidationError of the rest framework itself, it by default already returns the status 400.

import
```py
from rest_framework.exceptions import ValidationError
```
Example: https://github.com/weni-ai/weni-integrations-engine/blob/430dbb555554b7ac4820e5a5611792e70b839f98/marketplace/applications/views.py#L71

This review also serves the following lines:
[25](https://github.com/weni-ai/weni-integrations-engine/pull/154/files#diff-3b72502ca6203dab01e97e75b8ac7e87ca33eb36e865922488aa00d8cb17d1d5R25)
[54](https://github.com/weni-ai/weni-integrations-engine/pull/154/files#diff-3b72502ca6203dab01e97e75b8ac7e87ca33eb36e865922488aa00d8cb17d1d5R54)
[59](https://github.com/weni-ai/weni-integrations-engine/pull/154/files#diff-3b72502ca6203dab01e97e75b8ac7e87ca33eb36e865922488aa00d8cb17d1d5R59)